framework/signature-inference-scripts/infer-and-annotate.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/sh
+
+# This script runs the Checker Framework's signature inference
+# iteratively on a program, adding type annotations on the program, until the
+# .jaif files from one iteration are the same as the .jaif files from the
+# previous iteration.
+
+# This script receives as arguments:
+# 1. Processor's name.
+# 2. Classpath (target project's classpath).
+# 3. Extra processor arguments which will be passed to the checker.
+# 4. List of paths to .java files in a program.
+
+# Example of usage:
+#./infer-and-annotate.sh org.checkerframework.common.value.ValueChecker \
```

Comment:
I made it a concrete example using both "LockChecker,NullnessChecker" for plume-lib. The processors need to be quoted and separated by commas, without space. I also showed an example of the find command. I added the example to the manual too.